tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -231,11 +231,34 @@ module.exports = {
 
       const fileCheckbox =
         this.getFileRowSelectorByFileName(path) + this.elements.checkboxInFileRow.selector
-
       await this.toggleCheckbox(enableOrDisable, fileCheckbox, 'xpath')
 
       return this
     },
+    /**
+     *
+     * @param {string} path
+     * @param {boolean} status
+     */
+    isResourceSelected: async function(path, status) {
+      await this.waitForFileVisible(path)
+      const fileCheckbox =
+        this.getFileRowSelectorByFileName(path) + this.elements.checkboxInFileRow.selector
+      await this.api.element('xpath', fileCheckbox, result => {
+        if (!result.value.ELEMENT) {
+          return
+        }
+        this.api.elementIdSelected(result.value.ELEMENT, result => {
+          assert.strictEqual(
+            result.value,
+            status,
+            status === result
+              ? path + ' is expected to be selected but is not selected'
+              : path + ' is expected not to be selected but is selected'
+          )
+        })
+      })
+    },
```

Comment:
It's better idea to move assertions away from page objects as we discussed a while back